worldline/lending/.htaccess

```
@@ -0,0 +1,22 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+
+RewriteEngine on
+
+
+#Rewrite rules for lending vocabulary extension
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^lending$ https://worldline.github.io/nodejs-training/ [R=303,NE,L]

```

Pattern is relative to current dir, perhaps you mean to match https://w3id.org/worldline/lending instead of https://w3id.org/worldline/lending/lending?
```suggestion
RewriteRule ^$ https://worldline.github.io/nodejs-training/ [R=303,NE,L]
```